test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -12,7 +12,27 @@ namespace Microsoft.AspNetCore.Localization.FunctionalTests
     public class LocalizationTest
     {
         private static readonly string _applicationPath = Path.Combine("test", "LocalizationWebsite");
-        
+
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
So far as I can tell the idea is to avoid running CLR on Non-windows. But there seems to be a lot of duplication, I can file an issue to clean all of the tests in this format up at once.